FNPlugin/Collectors/SolarWindCollector.cs

```diff
@@ -18,23 +18,24 @@ class SolarWindCollector : FNResourceSuppliableModule
         public float fLastPowerPercentage;
 
         // Part properties
```

Comment:
Is your comment about the line 21? Github shows 18 to 20, so I am not certain if I understand correctly :) If yes, the escape character \xB creates superscript when used in a string.